src/util/fabric/theme.test.ts

```diff
@@ -1,5 +1,7 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
-jest.mock('office-ui-fabric-react/lib/Styling');
+jest.mock('office-ui-fabric-react/lib/Styling', () => ({
+  loadTheme: jest.fn(),
+}));
```

Comment:
This file now just re-exports `@uifabric/styling`, and Jest wasn't processing it properly.